src/solvers/dgsem_tree/container_viscous_2d.jl

```diff
@@ -0,0 +1,69 @@
+mutable struct ViscousContainer2D{uEltype <: Real}
```

Comment:
Am I right to assume that this is essentially the same as for 1D but extended to 2D?

Reply:
Hm, in some sense yes, since we have the same fields but of different dimensions, i.e., for 1D we have for instance 
`u_transformed::Array{uEltype, 3}` and for 2D `u_transformed::Array{uEltype, 4}`

Reply:
Yes, of course. I just wanted to confirm that there is no new logic/fields introduced that would need to be reviewed separately. Thanks!